src/indexPage/react-components/logo.jsx

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
-import { createUrl } from '../../common/utils/tools';
+import {createUrl, isBinaryDomain} from '../../common/utils/tools';
 
-const Logo = () => (
-    <a href={createUrl({ path: '', isNonBotPage: true })} target="blank" id="logo">
+const Logo = () => {
```

Comment:
there is no need to throw a variable inside the component if the condition is to display the component or not
you can use it like this:
`const Logo = () => (<a>...</a>>)`
and then where you use it write `{isBinaryDomain && <Logo/>}`

Reply:
When I met this place first time I was a little bit confused can we use logic here or not. But it works. Thanks 👍 

<img width="964" alt="Screenshot 2023-01-24 at 14 22 49" src="https://user-images.githubusercontent.com/103182473/214279263-f14d2333-6c5a-4ed3-a429-1cca67807261.png">